Tests/MSFT_xVMHyper-V/xVMHyper-V.Tests.ps1

```diff
@@ -29,6 +29,9 @@ Describe 'xVMHyper-V' {
         function Set-VMNetworkAdapter { }
         function Get-VMFirmware { }

```

Comment:
There should be a tests that verify that test-target resource passes/fails appropriately based on the state of EnableGuestService